gm4_area_markers/data/area_markers/functions/main.mcfunction

```diff
@@ -0,0 +1,20 @@
+#run from pulse_check
```

Comment:
This file contains 9 global `@e` selectors. This is unacceptable. 
4 of these all match `@e[type=minecraft:area_effect_cloud, tag=gm4_marker]`; These should all be in a function `process_marker` or something similar.
3 of these match `@e[type=minecraft:item, name="Player Head"]; These ones could go in `process_item` or something.

Reply:
K, lemme do that

Reply:
2nd part now is a thing but I have to go now so bye

Reply:
Done!